monkestation/code/modules/power/rbmk/rbmk_computers.dm

```diff
@@ -0,0 +1,344 @@
+//Controlling the reactor.
+/obj/machinery/computer/reactor
+	name = "Reactor control console"
+	desc = "Scream"
```

Comment:
scream

Reply:
Yeah I have no clue why NSV has it that way